lightning/src/events/mod.rs

```diff
@@ -1797,10 +1801,26 @@ impl Writeable for Event {
 			},
 			&Event::HTLCHandlingFailed { ref prev_channel_id, ref failure_type, ref failure_reason } => {
 				25u8.write(writer)?;
+
+				// The [`HTLCHandlingType::UnknownNextPeer`] variant is deprecated, but we want to
```

Comment:
Doc link needs updating. I believe the brackets don't do anything here unfortunately since it's in a comment within a method